independent-publisher-connectors/Infura Ethereum/apiDefinition.swagger.json

```diff
@@ -0,0 +1,258 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Infura Ethereum",
+    "description": "The Infura Ethereum Connector uses the Infura JSON-RPC API to access the Ethereum Blockchain.",
+    "version": "1.0",
+    "contact": {
+      "name": "Sebastian Zolg",
+      "url": "https://sebastianzolg.de",
+      "email": "sebastian.zolg@leantify.com"
+    }
+  },
+  "host": "mainnet.infura.io",
+  "basePath": "/v3",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/eth_gasPrice": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "result": {
+                  "type": "integer",
+                  "description": "Result",
+                  "title": "Result",
+                  "x-ms-visibility": "important",
+                  "format": "int32"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Gas Price",
+        "description": "Returns the current price per gas in wei.",
+        "operationId": "Eth_gasPrice",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "jsonrpc": {
+                  "type": "string",
+                  "description": "jsonrpc",
+                  "title": "",
```

Comment:
@sebastianzolg Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
Removed all empty `title: ""` properties.